chronograf.go

```diff
@@ -32,12 +33,13 @@ func (e Error) Error() string {
 type Logger interface {
 	Debug(...interface{})
 	Info(...interface{})
-	Warn(...interface{})
```

Comment:
I think we could have warnings in the future, yes.  I just slimmed this down to just what we are currently using.

To me warnings are used to "forecast" that an error will occur in the future.  For example, "Warning: you are at 95% disk usage. You will crash soon".

The other usage of warning would be fore deprecations: "Warning: deprecating in version 9000"

As for unreachable data node, is this a normal thing that can happen that is recoverable?  If so, then info. 

About handling next node sorts of information, sometimes that type of information is important in aggregate and thus is a metric.  For example, perhaps 100% of the time we are using next node, because the first node is not reachable.